### PR TITLE
Create login page files

### PR DESCRIPTION
**Pull request by chloesw**
change layout of login page and add sign up page files

### CODE REVIEW

File: research/frontend/css/index.css [hunk 1]
```diff
@@ -2,7 +2,12 @@ html,
 body {
   width: 100%;
```
Comment (shberry) [inline research/frontend/css/index.css:3]: Usually it's good to list the css properties in lexicographical order (alphabetical order in our case)

File: research/frontend/css/index.css [hunk 2]
```diff
@@ -2,7 +2,12 @@ html,
...
   margin: 0 auto;
-  background-color: #f7f7f7;
+  background: #e9e9e9;
+  color: #666666;
```
Comment (shberry) [inline research/frontend/css/index.css]: How about just #666? I think this will work too

File: research/frontend/css/index.css [hunk 3]
```diff
@@ -2,7 +2,12 @@ html,
...
+  font-family: 'RobotoDraft', 'Roboto', sans-serif;
+  font-size: 14px;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
```
Comment (shberry) [inline research/frontend/css/index.css]: in .qc-logo-icon, you can merge the two margin in one property. Maybe 'margin: 20px 0' ?

File: research/frontend/css/index.css [hunk 4]
```diff
@@ -36,6 +41,13 @@ nav {
   padding-top: 30px;
 }
 
+.feed-wrapper {
+  display: flex;
```
Comment (shberry) [inline research/frontend/css/index.css]: You'll need to add 'display: -webkit-flex;' for Safari and 'display: -ms-flexbox' for IE suport

File: research/frontend/css/login.css [hunk 5]
```diff
@@ -1,33 +1,27 @@
...
+  flex-direction: column; 
+  justify-content: center;
 }
 
-form {
-  padding: 0px;
-  border-radius: 3px;
-  box-sizing: border-box;
+.card-panel {
+  margin-top:160px;
```
Comment (shberry) [inline research/frontend/css/login.css]: nit: space after :

File: research/frontend/css/sign-up.css [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+.container {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    justify-content: center;
+}
+
+.card-panel {
+    margin-top:160px;
+  }
```
Comment (shberry) [inline research/frontend/css/sign-up.css]: remove extra space before }

File: research/frontend/sign-up.html [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+                            <input id="password" type="password">
+                            <label for="password">Password</label>
+                        </div>
+                    </div>
+                    <div class="center">
+                        <button class="btn amber waves-effect waves-light right" type="submit" name="action" onclick="location.href='login.html'">Submit
+                            <i class="mdi-content-send right"></i>
+                        </button>
+                    </div>
+
```
Comment (shberry) [inline research/frontend/sign-up.html]: remove extra line?

File: research/frontend/sign-up.html [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+                </form>
+            </div>
+        </div>
+    </div>
+
```
Comment (shberry) [inline research/frontend/sign-up.html]: same here

File: research/frontend/sign-up.html [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+<body>
+    <div class="container ">
+        <div class="card-panel z-depth-4 center">
+            <h4 class="header2">Register</h4>
+            <div class="row">
+                <form id="sign-up-form" class="col s12" method="post">
+                    <div class="row">
+                        <div class="input-field col s6">
+                            <input id="first_name" type="text">
```
Comment (shberry) [inline research/frontend/sign-up.html]: no closing tag? or <input xxxx /> will do

File: research/frontend/css/sign-up.css [hunk 10]
```diff
@@ -0,0 +1,21 @@
+body {
```
Comment (shberry) [inline research/frontend/css/sign-up.css:2]: I see a lot of common styles between css files. Usually it's preferred to have a common.css that holds basic styles and let each css file overwrite it. But I'm keeping it this way :) Up to you!

**Review by shberry — COMMENTED** [review]
Left a few comments while skimming through. Will take a closer look at the next round :) Looks good, thanks!

File: research/frontend/sign-up.html (hunk 9 above)
Comment (chloesw) [reply research/frontend/sign-up.html]: do you mean the input tag?

Comment (shberry) [reply research/frontend/sign-up.html]: oh my bad. You only need closing tag for XHTML. In HTML, <input> tag has no end tag :)

Comment (chloesw) [reply research/frontend/sign-up.html]: Oh i see! Thanks u sooooo much >_<

**Pull request merged**